src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
you need to add modificators for calendar and write scss func for position and hide entire items
(must looks like calendar--start-day--wed calendar--month-length--30)

Reply:
but i dont know how to write func